### ISSUE TITLE
Better liquid synergies with Nitroglycerine

### ISSUE THREAD

**Issue by RikouHiyama** [studio-minus/ppg-bugs-and-requests#1486]
**Feature description**
The ability to mix Nitroglycerine with various other liquids to alter/improve its effect

**Feature purpose**
Currently, (imo) Nitroglycerine is a very boring liquid, despite being a volatile explosive. It has a weak, low-radius explosion, and can't be modified through mixture with other liquids. Besides an explosion rework (which would likely take much more time), I believe that an update allowing for the creation of different nitroglycerine mixtures with effects relevant to their ingredients would not only breathe life into Nitroglycerine, but the liquid system as a whole.

As example, placing Oil into Nitroglycerine could cause the explosion to ignite within the radius. Pink serum could make a Nitroglycerine explosion happen multiple times (or simply increase its radius), and Tritium could turn a Nitroglycerine container into a liquid nuke, to produce a very powerful explosion.

**Additional context**
Included is a video of me slightly disappointed by the Liquid Nuke not existing: https://www.youtube.com/watch?v=e5rMHznl948

**Comment by DindinYT37** [comment]
Agree, but I'd rather have it be like a liquid mini-nuke. Since we already have a nuke and I think a nuke with a smaller radius would be great to have in the game.
